src/main/java/org/opentripplanner/routing/location/TemporaryStreetLocation.java

```diff
@@ -31,7 +32,7 @@ public void addIncoming(Edge edge) {
     @Override
     public void addOutgoing(Edge edge) {
         if (edge instanceof TemporaryEdge) {
-            if (endVertex) {
+            if (endVertex && !(edge instanceof TemporaryDropoffVehicleEdge)) {
                 throw new UnsupportedOperationException("Can't add outgoing edge to end vertex");
```

Comment:
Nie wiem, czy nie lepiej byłoby usunąć to rzucanie wyjątkiem całkowicie.

Reply:
Zmniejsza jednak szanse, że ktoś zrobi głupotę i tego nie wykryjemy

Reply:
Jak rozumiem problem jest spowodowany tym, że ta krawędź jest (end -> end). Jak bardzo byłoby skomplikowane dodanie wierzchołka nowej klasy np. `VehicleRentingVertex`  i zrobienie krawędzi (Vrv -> end).
Wiem, że bardziej to komplikuje srpawę, ale unikamy wtedy takich hacków które później nie wiadomo skąd się wzięły.

Reply:
To jest do przemyślenia, temat na pewno wypłynie jak spróbujemy ugryźć rzutowanie pojazdów w faktycznym miejscu postoju, a nie w najbliższym skrzyżowaniu. Wolałbym tego nie ruszać w tym tasku.